src/NuGet.Core/NuGet.PackageManagement/PublicAPI.Unshipped.txt

```diff
@@ -4,4 +4,4 @@ const NuGet.ProjectManagement.ProjectBuildProperties.TargetFrameworkIdentifier =
 const NuGet.ProjectManagement.ProjectBuildProperties.TargetFrameworkProfile = "TargetFrameworkProfile" -> string
 const NuGet.ProjectManagement.ProjectBuildProperties.TargetFrameworkVersion = "TargetFrameworkVersion" -> string
 const NuGet.ProjectManagement.ProjectBuildProperties.TargetPlatformMoniker = "TargetPlatformMoniker" -> string
-static NuGet.ProjectManagement.MessageLevelExtensions.ToLogLevel(this NuGet.ProjectManagement.MessageLevel messageLevel) -> NuGet.Common.LogLevel
\ No newline at end of file
+static NuGet.ProjectManagement.MessageLevelExtensions.ToLogLevel(this NuGet.ProjectManagement.MessageLevel messageLevel) -> NuGet.Common.LogLevel
```

Comment:
this is a NoOp?

Reply:
This is not mine, I think I saved this file while rebasing and it added a newline at the end

Reply:
If that's the case, I think you should revert this. 

Sample command: 

> git checkout dev -- src/NuGet.Core/NuGet.PackageManagement/PublicAPI.Unshipped.txt 

Should revert only that specific file.